From 9d6cbbd87a40cba7c611bbcb74d749edebd70c38 Mon Sep 17 00:00:00 2001
From: "yang.wang" <yang.wang@deepcomputing.io>
Date: Tue, 18 Mar 2025 10:35:43 +0800
Subject: [PATCH 241/540] defconfig: fml13v03_defconfig: Fix compilation
 warnings

arch/riscv/configs/fml13v03_defconfig:816:warning: override: reassigning to symbol TMPFS_POSIX_ACL
arch/riscv/configs/fml13v03_defconfig:886:warning: override: reassigning to symbol EFI_VARS_PSTORE
arch/riscv/configs/fml13v03_defconfig:887:warning: override: reassigning to symbol EFI_BOOTLOADER_CONTROL
arch/riscv/configs/fml13v03_defconfig:888:warning: override: reassigning to symbol EFI_CAPSULE_LOADER
arch/riscv/configs/fml13v03_defconfig:889:warning: override: reassigning to symbol EFI_TEST
arch/riscv/configs/fml13v03_defconfig:890:warning: override: reassigning to symbol EFI_COCO_SECRET

Signed-off-by: yang.wang <yang.wang@deepcomputing.io>
---
 arch/riscv/configs/fml13v03_defconfig | 9 ---------
 1 file changed, 9 deletions(-)

diff --git a/arch/riscv/configs/fml13v03_defconfig b/arch/riscv/configs/fml13v03_defconfig
index 2e69a9e8adc33..87f975b2cdce2 100755
--- a/arch/riscv/configs/fml13v03_defconfig
+++ b/arch/riscv/configs/fml13v03_defconfig
@@ -317,7 +317,6 @@ CONFIG_FW_LOADER_COMPRESS=y
 CONFIG_FW_LOADER_COMPRESS_ZSTD=y
 CONFIG_FIRMWARE_MEMMAP=y
 CONFIG_SYSFB_SIMPLEFB=y
-CONFIG_EFI_VARS_PSTORE=m
 CONFIG_EFI_BOOTLOADER_CONTROL=m
 CONFIG_EFI_CAPSULE_LOADER=m
 CONFIG_EFI_TEST=m
@@ -793,7 +792,6 @@ CONFIG_INTERCONNECT=y
 CONFIG_EXT4_FS=y
 CONFIG_EXT4_FS_POSIX_ACL=y
 CONFIG_EXT4_FS_SECURITY=y
-CONFIG_FAT_FS=y
 CONFIG_BTRFS_FS=m
 CONFIG_BTRFS_FS_POSIX_ACL=y
 CONFIG_FS_ENCRYPTION=y
@@ -813,7 +811,6 @@ CONFIG_NTFS_FS=m
 CONFIG_NTFS_RW=y
 CONFIG_TMPFS=y
 CONFIG_TMPFS_POSIX_ACL=y
-CONFIG_TMPFS_POSIX_ACL=y
 CONFIG_TMPFS_INODE64=y
 CONFIG_TMPFS_QUOTA=y
 CONFIG_HUGETLBFS=y
@@ -883,9 +880,3 @@ CONFIG_RCU_EQS_DEBUG=y
 # CONFIG_FTRACE is not set
 # CONFIG_RUNTIME_TESTING_MENU is not set
 CONFIG_MEMTEST=y
-CONFIG_EFI_VARS_PSTORE=m
-CONFIG_EFI_BOOTLOADER_CONTROL=m
-CONFIG_EFI_CAPSULE_LOADER=m
-CONFIG_EFI_TEST=m
-CONFIG_EFI_COCO_SECRET=y
-CONFIG_EFI_STUB=y
-- 
2.53.0

